client/src/components/ToolsClockFields.vue

```diff
@@ -57,5 +59,13 @@ export default {
       type: Boolean,
     },
   },
-};
+  setup() {
+    const padValue = (value: number | string) => {
+      return value < 10 ? `0${value}` : value;
+    }
+    return {
+      padValue,
```

Comment:
I don't quite get the function name

Reply:
it's based on [https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart)